payroll/models/hr_payslip.py

```diff
@@ -511,7 +511,7 @@ def _get_baselocaldict(self, contracts):
             line.code: line for line in self.input_line_ids if line.code
         }
         localdict = {
-            "payslip": Payslips(self.employee_id.id, self, self.env),
+            "payslips": Payslips(self.employee_id.id, self, self.env),
```

Comment:
Is this by mistake? Because if it isn't it will break people's rules who might be using it.

Reply:
Actually i discovered this when doing this PR and i think is a bug. Since the "payslip" object, should refer to payslip object, not to the browsable version of it (see Payslips in base_browsable.py). 

So it was never the payslip object, it was the Payslips object with "payslip" name.

Reply:
See below answers to understand more.